web/pgadmin/static/js/components/PgReactDataGrid.jsx

```diff
@@ -119,7 +119,7 @@ export function CustomRow({inTest=false, ...props}) {
 
   return (
     <Row {...props} onKeyDown={handleKeyDown} onCellClick={onCellClick} onCellDoubleClick={onCellDoubleClick}
-      selectCell={(row, column)=>props.selectCell(row, column)} aria-selected={isRowSelected}/>
+      selectCell={(row, column)=>props.selectCell(row, column)} aria-selected={isRowSelected} rangeSelectionMode={true}/>
```

Comment:
PgReactDataGrid is a common component and rangeSelectionMode true is not required here. rangeSelectionMode is only required in query tool grid.

Reply:
thanks for looking into this @adityatoshniwal - that was careless on my part. have updated so that it is only enabled from query tool component via `prop.enableRangeSelection`